docker-environment-driver/src/main/resources/isolatedDockerConfiguration/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM fedora
+MAINTAINER Marek Novotny <mnovotny@redhat.com>
+
+# install java and zip
+RUN yum -y update && yum clean all
+RUN yum -y install tar gzip java java-devel && yum clean all
+
+# set JAVA_HOME
+ENV JAVA_HOME /usr/lib/jvm/java-openjdk
+
+## jenkins instalation
+ENV JENKINS_HOME /var/jenkins_home
```

Comment:
Why we are not using standard jenkins installation directory `/var/lib/jenkins` ?